direct.scrubsv2.json

```diff
@@ -1,7 +1,8 @@
 {
-    "name" : "[COLOR orange]Scrubs v2[/COLOR]",
-    "plugin" : "plugin.video.scrubsv2",
-    "priority" : 500,
-    "play_movie" : "plugin://plugin.video.scrubsv2/?action=play&amp;title={title_+}&amp;year={year}&amp;imdb={imdb}&amp;meta=%7B%22rating%22%3A+%22%22%2C+%22code%22%3A+%22{imdb}%22%2C+%22tmdb%22%3A+%22{id}%22%2C+%22imdb%22%3A+%22{imdb}%22%2C+%22year%22%3A+%22{year}%22%2C+%22duration%22%3A+%22%22%2C+%22plot%22%3A+%22{plot}%22%2C+%22votes%22%3A+%22%22%2C+%22title%22%3A+%22{title_+}%22%2C+%22fanart%22%3A+%22{fanart}%22%2C+%22tagline%22%3A+%22%22%2C+%22writer%22%3A+%22%22%2C+%22next%22%3A+%22%22%2C+%22poster%22%3A+%22{poster}%22%2C+%22mediatype%22%3A+%22movie%22%2C+%22director%22%3A+%22%22%2C+%22studio%22%3A+%22%22%2C+%22genre%22%3A+%22%22%2C+%22metacache%22%3A+true%2C+%22premiered%22%3A+%22{premiered}%22%2C+%22originaltitle%22%3A+%22{title_+}%22%2C+%22cast%22%3A+%5B%5B%22%22%5D%5D%2C+%22mpaa%22%3A+%22%22%2C+%22trailer%22%3A+%22plugin%3A%2F%2Fplugin.video.scrubsv2%2F%3Faction%3Dtrailer%26name%3D{title_+}%2B%2528{year}%2529%22%7D&amp;t={now}",
-    "play_episode" : "plugin://plugin.video.scrubsv2/?action=play&amp;title={title_+}&amp;year={year}&amp;imdb={imdb}&amp;tvdb={id}&amp;season={season}&amp;episode={episode}&amp;tvshowtitle={clearname_+}&amp;premiered={firstaired}&amp;meta=%7B%22rating%22%3A+%22%22%2C+%22code%22%3A+%22{imdb}%22%2C+%22tmdb%22%3A+{tmdb}%2C+%22imdb%22%3A+%22{imdb}%22%2C+%22year%22%3A+%22{year}%22%2C+%22duration%22%3A+%22%22%2C+%22plot%22%3A+%22{plot}%22%2C+%22votes%22%3A+%22%22%2C+%22thumb%22%3A+%22{thumbnail}%22%2C+%22title%22%3A+%22{title_+}%22%2C+%22tvdb%22%3A+%22{id}%22%2C+%22label%22%3A+%22{title_+}%22%2C+%22season%22%3A+%22{season}%22%2C+%22status%22%3A+%22%22%2C+%22poster%22%3A+%22{poster}%22%2C+%22tvshowtitle%22%3A+%22{clearname_+}%22%2C+%22mediatype%22%3A+%22episode%22%2C+%22studio%22%3A+%22%22%2C+%22genre%22%3A+%22%22%2C+%22banner%22%3A+%22{thumbnail}%22%2C+%22episode%22%3A+%22{episode}%22%2C+%22premiered%22%3A+%22{firstaired}%22%2C+%22fanart%22%3A+%22{fanart}%22%2C+%22trailer%22%3A+%22plugin%3A%2F%2Fplugin.video.scrubsv2%2F%3Faction%3Dtrailer%26name%3D{clearname_+}%22%7D&amp;t={now}"
+  "name" : "Scrubs V2", 
```

Comment:
I see that in this case (also for [THE CREW](https://github.com/a4k-openproject/json.openplayers/pull/106/files#diff-16ab15b0d89507f439acf5f99461ad54921f6e32e369ace3a4e61762fab9be74R2) and [Genocide](https://github.com/a4k-openproject/json.openplayers/pull/106/files#diff-bcee13fb05839d2639477beb372fdc314913677d76d4d0682ef0e903d1d821baR2)) you've removed the color, but decided to leave it in on a few others, and even fix it in Genocide... Is there any logic to these?